test/sourcemaps/samples/compile-option-dev/input.svelte

```diff
@@ -0,0 +1,15 @@
+<h1>Testing Styles</h1>
+<h2>Testing Styles 2</h2>
+<div>Testing Styles 3</div>
+<script>export const b = 2;</script>
+<style>
+	h1 {
+		--replace-me-once: red;
+	}
+	h2 {
+		--replace-me-twice: green;
+	}
+	div {
+		--keep-me: blue;
+	}
+</style>
```

Comment:
perhaps add a newline at the end of this file for consistency (eslint doesn't run on `.svelte` files in this repo, so wouldn't catch this one, but it's generally something we try to follow)